specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/managedHsm.json

```diff
@@ -1189,21 +1206,24 @@
           "x-ms-enum": {
             "name": "NetworkRuleAction",
             "modelAsString": true
-          }
+          },
+          "default": "Allow"
         },
         "ipRules": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/MHSMIPRule"
           },
-          "description": "The list of IP address rules."
+          "description": "The list of IP address rules.",
+          "default": []
```

Comment:
ARM template schemas generation failed, due to a known issue - the pipeline doesn't support non-string type default value. Please revert the `default: []` changes, otherwise keyvault schemas cannot be generated.

For S360 issues, please submit exception request to suppress them: https://aka.ms/swaggerkpiex

Reply:
Addressed and requested for an exception.